### PR TITLE
Add/additional font size

### PR DESCRIPTION
**Pull request by pbking**
**Description**

This change adds an additional font size.  (XX-Large)

However, since a SMALLER font size is what is needed this shifts ALL of the font sizes UP

The value of 'small' becomes the value of 'medium' and is assigned as the default size.
The value of 'medium' becomes the value of 'large'
The value of 'large' becomes the value of 'x-large'
The value of 'x-large' becomes the value of the NEW size 'xx-large'

And SMALL becomes a NEW value of `clamp(0.813rem, 0.688rem)`
(I'm not tied to this value, it just seemed to hit most of what was needed based on the explicit values used in existing patterns)

Additionally the font size values have been added to the readme.md file for easy reference.

**Screenshots**

There are no visible changes to the theme, patterns, layout, etc (assuming I made all of the changes correctly...)

### CODE REVIEW

File: theme.json [hunk 1]
```diff
@@ -253,41 +253,50 @@
 				}
 			],
 			"fontSizes": [
+				{
+					"fluid": {
+						"max": "0.813rem",
+						"min": "0.688rem"
```
Comment (MaggieCabrera) [inline theme.json:259]: I would keep this one non fluid since it's small as it is. Just let it be 0.8 on any screen size

**Comment by MaggieCabrera** [comment]
This is what I would have done too, thank you @pbking! I would love feedback on this from both design and a11y perspective. I agree the new font size is a bit too small for comfort, but I think if we keep it on just a few use cases like the post meta, it should be fine?

**Comment by richtabor** [comment]
I'm looking now.

**Comment by pbking** [comment]
Closing in lieu of #268

**Pull request closed without merging**